packages/contentstack-import/src/import/modules-js/extensions.js

```diff
@@ -92,6 +92,15 @@ module.exports = class ImportExtensions {
         .catch(function (error) {
           // error while importing extensions
           fileHelper.writeFileSync(extFailsPath, self.fails);
+          let message_content_type = '';
+          if (error.request !== undefined && JSON.parse(error.request.data).extension.title !== undefined) {
+            if (JSON.parse(error.request.data).extension.title) {
+              message_content_type =
+                ' Failed extension: ' + JSON.parse(error.request.data).extension.title;
+            }
+            error.errorMessage = error.errorMessage + message_content_type;
+          }
+          log(self.config, formatError(error.errorMessage), 'error');
```

Comment:
@cs-raj Could you please check to see if we need this? It has already been addressed above.

Reply:
@aman19K  changes are done and removed the redundant code